### PR TITLE
Update whats-new.mdx

### PR DESCRIPTION
**Pull request by vavalos5**
[Deploy Preview](https://deploy-preview-2081--keen-mayer-a86c8b.netlify.app/whats-new?q=whats)

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

### CODE REVIEW

File: aries-site/src/pages/whats-new.mdx [hunk 1]
```diff
@@ -52,6 +52,31 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+)
+- **Button Guidance**.
+Guidance on left or right aligned buttons on forms, modals, layers and wizards [updated in Figma.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=4071%3A8993)
+
+### Stay current with the latest changes
+- **Icons** - [Removed stroke version of icons from Figma](https://www.figma.com/file/G99KGffGLFKICy8xRs51lW/HPE-Icons?node-id=261%3A0) component library to limit to outline versions only so icons are correctly sized.
+- **Tables** - Guidance and examples on how to show [states and statuses within a table.](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-DataTable?node-id=7210%3A147386)
+- **Checkbox** - [Align checkbox label when multi-line.](https://github.com/grommet/grommet-theme-hpe/pull/222)
+- **FileInput** - Added an example with a value that needs validation on [Design System site](https://design-system.hpe.design/components/fileinput?q=file#fileinput-validation)and [Figma.](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=1164%3A43348)
```
Comment (jcfilben) [inline aries-site/src/pages/whats-new.mdx:75]:
```suggestion
- **FileInput** - Added an example with a value that needs validation on [Design System site](https://design-system.hpe.design/components/fileinput?q=file#fileinput-validation) and [Figma](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=1164%3A43348).
```
Missing a space between 'Design System Site' and 'and'.

File: aries-site/src/pages/whats-new.mdx [hunk 2]
```diff
@@ -52,6 +52,31 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- **Grommet** - [Released Grommet 2.19.0](https://github.com/grommet/grommet/releases/tag/v2.19.0)
+- **Grommet-Theme-HPE** - [Released Grommet-theme-hpe v2.3.0](https://github.com/grommet/grommet-theme-hpe/releases/tag/v2.3.0)
+- **Grommet-Icons** - [Released Grommet-Icons v4.7.0](https://github.com/grommet/grommet-icons/releases/tag/v4.7.0)
```
Comment (jcfilben) [inline aries-site/src/pages/whats-new.mdx:78]: Should the releases should get their own 'Releases' section?

Comment (vavalos5) [reply aries-site/src/pages/whats-new.mdx:78]: @britt6612 What do you think? That might be good

Comment (britt6612) [reply aries-site/src/pages/whats-new.mdx:78]: good idea I think thats good

Comment (britt6612) [reply aries-site/src/pages/whats-new.mdx:80]: I moved this over to its own section

File: aries-site/src/pages/whats-new.mdx [hunk 3]
```diff
@@ -52,6 +52,33 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- **Figma Guide** - A fundamental and advanced functionality of Figma to help you understand and leverage the HPE Design System 
+in your [design process.](https://www.figma.com/proto/ehx1youYYwuaGasfwTbkNK/Figma-Guide?page-id=425%3A284&node-id=846%3A1435&viewport=241%2C48%2C0.03&scaling=min-zoom&starting-point-node-id=644%3A590)
+
+### "Big Things" progress updates
+Here are the latest developments regarding roadmap items:
+- **NameValueList**. Now available in [Figma](https://www.figma.com/file/Ny7rYK5gI5VO4PtZsq6vGC/HPE-Name-Value-List?node-id=2%3A93) and [Design System site.](https://design-system.hpe.design/components/namevaluelist?q=name)
+- **Navigation**. Guidance and examples for [Drill Down Navigation on Card, List and Tables.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9622
+)
+- **Button Guidance**.
+Guidance on left or right aligned buttons on forms, modals, layers and wizards [updated in Figma.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=4071%3A8993)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:69]:
```suggestion
Left aligned? Right aligned? Read the latest guidance for [how buttons should be aligned](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=4071%3A8993) in various contexts.
```

File: aries-site/src/pages/whats-new.mdx [hunk 4]
```diff
@@ -52,6 +52,33 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+
+### Stay current with the latest changes
+- **Icons** - [Removed stroke version of icons from Figma](https://www.figma.com/file/G99KGffGLFKICy8xRs51lW/HPE-Icons?node-id=261%3A0) component library to limit to outline versions only so icons are correctly sized.
+- **Tables** - Guidance and examples on how to show [states and statuses within a table.](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-DataTable?node-id=7210%3A147386)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:73]:
```suggestion
- **Tables** - Presenting state and statuses in table data can occupy valuable real-estate. View these recommendations for displaying [states and statuses within a table](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-DataTable?node-id=7210%3A147386).
```

File: aries-site/src/pages/whats-new.mdx [hunk 5]
```diff
@@ -52,6 +52,33 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- **Checkbox** - [Align checkbox label when multi-line.](https://github.com/grommet/grommet-theme-hpe/pull/222)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:74]:
```suggestion
- **Checkbox** - Checkboxes may now be [vertically aligned](https://github.com/grommet/grommet-theme-hpe/pull/222) when accompanied by a multi-line label such as your favorite "I accept the Terms of Use and Privacy Policy".
```

File: aries-site/src/pages/whats-new.mdx [hunk 6]
```diff
@@ -52,6 +52,33 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- **FileInput** - Added an example with a value that needs validation on [Design System site](https://design-system.hpe.design/components/fileinput?q=file#fileinput-validation) and [Figma](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=1164%3A43348).
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:75]:
```suggestion
- **FileInput** - Following up on an Office Hours request, examples for how file input validation should appear have been added to the [Design System site](https://design-system.hpe.design/components/fileinput?q=file#fileinput-validation) and [Figma](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=1164%3A43348).
```

File: aries-site/src/pages/whats-new.mdx [hunk 7]
```diff
@@ -52,6 +52,33 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+
+### Latest Releases
+- **Grommet** - [Released Grommet 2.19.0](https://github.com/grommet/grommet/releases/tag/v2.19.0)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:78]:
```suggestion
- **Grommet** - Released [Grommet 2.19.0](https://github.com/grommet/grommet/releases/tag/v2.19.0).
```

In the spirit of "why should I care?" This should be accompanied by any noteworthy features included in the release.

File: aries-site/src/pages/whats-new.mdx [hunk 8]
```diff
@@ -52,6 +52,33 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- **Grommet-Theme-HPE** - [Released Grommet-theme-hpe v2.3.0](https://github.com/grommet/grommet-theme-hpe/releases/tag/v2.3.0)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:79]:
```suggestion
- **Grommet-Theme-HPE** - Released [Grommet-theme-hpe v2.3.0](https://github.com/grommet/grommet-theme-hpe/releases/tag/v2.3.0).
```

In the spirit of "why should I care?" This should be accompanied by any noteworthy features included in the release.

**Review by jcfilben — APPROVED** [review]
Looks good!

File: aries-site/src/pages/whats-new.mdx (hunk 2 above)
Comment (halocline) [reply aries-site/src/pages/whats-new.mdx:80]: RE: Grommet Icons

In the spirit of "why should I care?" This should be accompanied by any noteworthy features included in the release.

File: aries-site/src/pages/whats-new.mdx (hunk 7 above)
Comment (britt6612) [reply aries-site/src/pages/whats-new.mdx:78]: You got it! 👍 good point

File: aries-site/src/pages/whats-new.mdx [hunk 9]
```diff
@@ -52,6 +52,33 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## November 8 - November 19, 2021
+
+### Communications
+- **Office Hours** - A second, timezone-friendly option for office hours was held with the India team to ensure all questions were answered.
+- **Email** - An e-mail was sent to the India team that directly answered previously asked questions.
```
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/whats-new.mdx]: sounds good I will remove

File: aries-site/src/pages/whats-new.mdx [hunk 10]
```diff
@@ -52,6 +52,33 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## November 8 - November 19, 2021
+
+### Communications
+- **Office Hours** - A second, timezone-friendly option for office hours was held with the India team to ensure all questions were answered.
```
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/whats-new.mdx:58]: Kamlesh did not include a series so not sure if were going to have same data and time

[the remark this answers is not in the document]
Comment (taysea) [reply aries-site/src/pages/whats-new.mdx:58]: I have it recurring on my calendar for Thursday 7:30-8:30am PT

File: aries-site/src/pages/whats-new.mdx [hunk 11]
```diff
@@ -52,6 +52,29 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+
+### Stay current with the latest changes
+- **Tables** - Presenting state and statuses in table data can occupy valuable real-estate. View these recommendations for displaying [states and statuses within a table](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-DataTable?node-id=7210%3A147386).
+- **Checkbox** - Checkboxes may now be [vertically aligned](https://github.com/grommet/grommet-theme-hpe/pull/222) when accompanied by a multi-line label such as your favorite "I accept the Terms of Use and Privacy Policy".
+- **FileInput** - Following up on an Office Hours request, examples for how file input validation should appear have been added to the [Design System site](https://design-system.hpe.design/components/fileinput?q=file#fileinput-validation) and [Figma](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=1164%3A43348).
+
+### Latest Releases
+- **Grommet** - Released Grommet 2.19.0 which includes the NEW NameValueList component, Enhancement to DataTable, FileInput, FormField and RangeInput [Release Notes here](https://github.com/grommet/grommet/releases/tag/v2.19.0)
```
Comment (taysea) [inline aries-site/src/pages/whats-new.mdx:74]:
```suggestion
- **Grommet** - Released Grommet 2.19.0 which includes the NEW NameValueList component along with enhancements to DataTable, FileInput, FormField and RangeInput. For a comprehensive list of enhancements and fixes, see the [Release Notes](https://github.com/grommet/grommet/releases/tag/v2.19.0)
```

File: aries-site/src/pages/whats-new.mdx [hunk 12]
```diff
@@ -52,6 +52,29 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- **Grommet-Theme-HPE** Release included aligning Input, RangeInput to figma. Fixed DataTable pinned background to work in both light and dark modes.  - [Release Notes](https://github.com/grommet/grommet-theme-hpe/releases/tag/v2.3.0)
```
Comment (taysea) [inline aries-site/src/pages/whats-new.mdx:75]:
```suggestion
- **grommet-theme-hpe v2.3.0** - Release includes alignments of input dimensions and RangeInput to Figma, fixes to DataTable pinned background to work in both light and dark modes, and more.  For more details, see the [Release Notes](https://github.com/grommet/grommet-theme-hpe/releases/tag/v2.3.0).
```

File: aries-site/src/pages/whats-new.mdx (hunk 10 above)
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/whats-new.mdx:58]: oh good to know I dont have it

[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/whats-new.mdx:58]: Ill add the time

File: aries-site/src/pages/whats-new.mdx [hunk 13]
```diff
@@ -52,6 +52,29 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- **Grommet-Icons** - Grommet-Icons release includes the new HomeRounded Icon. [Released Grommet-Icons v4.7.0](https://github.com/grommet/grommet-icons/releases/tag/v4.7.0)
```
Comment (taysea) [inline aries-site/src/pages/whats-new.mdx:76]:
```suggestion
- **grommet-icons v4.7.0** - Release includes the NEW HomeRounded icon. See the [Release Notes](https://github.com/grommet/grommet-icons/releases/tag/v4.7.0) for details.
```

**Review by taysea — APPROVED** [review]

File: aries-site/src/pages/whats-new.mdx [hunk 14]
```diff
@@ -52,6 +52,30 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## November 8 - November 19, 2021
+
+### Communications
+- **Office Hours** - A second, office hours is now going to be held on Thursdays 7:30-8:30am PT, this office hours is open to all. Contact us if you would like an invite.
+-  A second, timezone-friendly option for office hours was held with the India team to ensure all questions were answered.
```
[the remark this answers is not in the document]
Comment (halocline) [reply aries-site/src/pages/whats-new.mdx:59]: Please remove.

File: aries-site/src/pages/whats-new.mdx [hunk 15]
```diff
@@ -52,6 +52,30 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## November 8 - November 19, 2021
+
+### Communications
+- **Office Hours** - A second, office hours is now going to be held on Thursdays 7:30-8:30am PT, this office hours is open to all. Contact us if you would like an invite.
```
[the remark this answers is not in the document]
Comment (halocline) [reply aries-site/src/pages/whats-new.mdx:58]:
```suggestion
- **Office Hours** - A second office hours will be held on Thursdays, 9:00 PM IST / 7:30 AM PST. This office hours is open to all. Contact us if you would like an invite.
```

File: aries-site/src/pages/whats-new.mdx [hunk 16]
```diff
@@ -52,6 +52,30 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+-  A second, timezone-friendly option for office hours was held with the India team to ensure all questions were answered.
+- **Figma Guide** - A [new guide]((https://www.figma.com/proto/ehx1youYYwuaGasfwTbkNK/Figma-Guide?page-id=425%3A284&node-id=846%3A1435&viewport=241%2C48%2C0.03&scaling=min-zoom&starting-point-node-id=644%3A590)) to level-up your Figma skills. Whether you are on-boarding or a Figma power user, there's content for you from fundamentals, to auto-layout, to interactive prototyping -- plus fun tips and tricks too!
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:60]:
```suggestion
- **Figma Guide** - A [new guide](https://www.figma.com/proto/ehx1youYYwuaGasfwTbkNK/Figma-Guide?page-id=425%3A284&node-id=846%3A1435&viewport=241%2C48%2C0.03&scaling=min-zoom&starting-point-node-id=644%3A590) to level-up your Figma skills. Whether you are onboarding or a Figma power user, there's content for you from fundamentals, to auto-layout, to interactive prototyping &mdash; plus fun tips and tricks too!
```

File: aries-site/src/pages/whats-new.mdx [hunk 17]
```diff
@@ -52,6 +52,30 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+
+### Stay current with the latest changes
+- **Tables** - Presenting state and statuses in table data can occupy valuable real-estate. View these recommendations for displaying [states and statuses within a table](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-DataTable?node-id=7210%3A147386).
+- **Checkbox** - Checkboxes may now be [vertically aligned](https://github.com/grommet/grommet-theme-hpe/pull/222) when accompanied by a multi-line label such as your favorite "I accept the Terms of Use and Privacy Policy".
+- **FileInput** - Following up on an Office Hours request, examples for how file input validation should appear have been added to the [Design System site](https://design-system.hpe.design/components/fileinput?q=file#fileinput-validation) and [Figma](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=1164%3A43348).
+
+### Latest Releases
+- **Grommet v2.19.0** - Release includes the NEW NameValueList component along with enhancements to DataTable, FileInput, FormField and RangeInput. For a comprehensive list of enhancements and fixes, see the [Release Notes](https://github.com/grommet/grommet/releases/tag/v2.19.0)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:75]:
```suggestion
- **grommet v2.19.0** - Release includes the NEW NameValueList component along with enhancements to DataTable, FileInput, FormField and RangeInput. For a comprehensive list of enhancements and fixes, see the [Release Notes](https://github.com/grommet/grommet/releases/tag/v2.19.0).
```

**Review by halocline — CHANGES_REQUESTED** [review]

File: aries-site/src/pages/whats-new.mdx (hunk 14 above)
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/pages/whats-new.mdx:59]: sorry thought I had removed prior

**Review by halocline — APPROVED** [review]

**Pull request merged**